lib/tasks/transcribe.js

```diff
@@ -265,7 +266,8 @@ class TaskTranscribe extends SttTask {
         }
     }
     // save dedicated bugname for each endpoint
-    ep.transcribe_bugname = `${bugname}_${Date.now()}`;
+    ep.transcribe_bugname = `${bugname}_${channel}_${this.bugname_counter}`;
```

Comment:
maybe just 
```
ep.transcribe_bugname = `${bugname}_${channel}_${this.bugname_counter++}`;
```

Reply:
No, we can't use this, 

first verb will have transcribe_bugname_1_0
LCC to move the call to new transcribe verb in listen will have same bug name again transcribe_bugname_1_0

in the meanwhile the first transcribe is still being closed, the 2nd transcribe save private data to same bug name again

Still Date.now() is a must to make sure every single time we have different bugname